datahub-web-react/src/graphql/lineage.graphql

```diff
@@ -42,6 +42,16 @@ fragment lineageNodeProperties on EntityWithRelationships {
         status {
             removed
         }
+        fineGrainedLineages {
```

Comment:
Is the aspect called fineGrainedLineages? 

Typically, the names of these top level fields we try to align with the aspect. Should we do that here?

Reply:
ahh yes you right. we do already have an `inputOutput` field above i'm seeing now - it looks like we deprecated it a while ago but I can un-deprecate the overall field and deprecate the individual fields inside of it (those are already deprecated as well it looks like)

Reply:
hmm i wonder why we originally deprecated.. i'm okay with undeprecation play